.gitignore

```diff
@@ -1,3 +1,5 @@
 userConfig.h
 *.bak
 *.swp
+.vscode
+IoTMQTTPanel/IoTMQTTPanel-rancilio_v2.3.0_v2.json
```

Comment:
Warum das?

Reply:
.vscode ist die Erweiterung für die Visual Studio Code IDE, ist ein versteckter Ordner wo Einstellungsdateien, in dem Fall zum Beispiel für die Node oder Pfade für Libraries drin stehen. IoT ist eine angepasste Version von mir zum rumspielen.